CHANGELOG.adoc

```diff
@@ -8,6 +8,8 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
 
 == [Unreleased]
 
+== [1.0.0-M1] - 2022-10-25
```

Comment:
2023-10-19
Year is wrong, and the date the M1 artifact was published was the 19th. Though the year is the more important part :)

Reply:
ahauhasuahau
thanks